python/ray/util/sgd/v2/tests/test_worker_group.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
Can we add a test for the expected behavior if there aren't enough resources available to show if it is expected to hang or fail?

Reply:
If there's not enough resources it will hang/autoscale (just like with any other actor). I can add this to the docstring.